projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -83,6 +84,31 @@ describe('AuthService', () => {
     });
   });
 
+  describe('The user observable', () => {
```

Comment:
Nit: if we are making reference to public properties it could help using something like 

```suggestion
  describe(`The 'user' observable`, () => {
```
or

```suggestion
  describe('The `user` observable', () => {
```
 
Same for `isAuthenticated` above.